colossalai/inference/core/request_handler.py

```diff
@@ -297,11 +297,13 @@ def search_tokens(self, generation_config: GenerationConfig, logits):
         Sample tokens for finished requests.
         """
         # do logit processor
-        # NOTE: need to decide the granularity to process logits (sequence or batch)
-        config_dict = generation_config.to_dict()
-        for type in ["top_k", "top_p", "min_p"]:
-            if type in config_dict and config_dict[type] is not None:
-                logits = logit_processor(type, logits, config_dict[type])
+        top_p = generation_config.top_p
+        top_k = generation_config.top_k
+
+        if top_k:
+            logits = logit_processor("top_k", logits, top_k)
+        if top_p:
+            logits = logit_processor("top_p", logits, top_p)
```

Comment:
Wonder how is the attribute `min_p` used to get worked. Is it a futile attribute so that we skip it here?